boards/arduino-zero/doc.txt

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
doesn't doxygen directly support *.md?

Reply:
Apparently not for automatically parsed/generated documentation (doxygen comment, .txt or .doc files). It does support md for explicitly included pages (mainpage.md, etc).

Reply:
And Doxygen supports markdown syntax in comments.

Reply:
Actually, adding e.g., "doc.md" to doxygen's FILE_PATTERNS makes it include plain markdown just fine.
It even parses ```@xxx``` doxygen patterns.

See https://gist.github.com/anonymous/2b1611e7e2f751c93c6067435814db46

I suggest changing all those doc.txt into doc.md as done in above gist, and make them proper markdown files (not the C comment stuff). That way our editor also understand the format.

Reply:
Very good to known. This opens interesting perspectives for improving the documentation (not only boards).

Reply:
cool. I'm scripting that. Push commit soon

Reply:
wait, I just tested it locally, enabling it will parse all .md files in the repo. It has some side effects

Reply:
use `doc.md`

Reply:
did it work for you @aabadie?
The files are being indexed, but I have this left menu now:

![](https://gist.githubusercontent.com/jia200x/daac639c022d5b085e0019b63eadbdb0/raw/e3abfc587997d158ceed274f225cf0d78f8cdf61/img.png)